Documentation/marconi.rst

```diff
@@ -0,0 +1,277 @@
+.. _compile_run_marconi:
+
+Marconi 100
+===========
+
+Marconi 100 is a distributed multi-GPU HPC system equipped with 4 Nvidia V100 GPUs
+and 2 IBM Power9 CPUs per node. This architecture usually comes with a pre-installed
+CUDA-Aware Spectrum-MPI. However, SeisSol cannot operate with Spectrum-MPI because 
+of GPU memory leaks caused by this MPI implementation. This part of documentation
+describes how to setup and configure OpenMPI 4.1.x together with UCX 1.10.x to operate
+on IBM Power and similar HPC systems. 
+
+Installing Main Libraries and Packages
+---------------------------------------
+
+1. Create the following launch file:
+
+::
+
+  $ touch $HOME/launch.sh
+  $ cat $HOME/launch.sh
+  #!/bin/bash
+
+  VERSION="openmpi4-1-1_ucx1-10-1"
+  export SEISSOL_INSTALL=$HOME/usr/local/${VERSION}
+
+  export PATH=$SEISSOL_INSTALL/bin:$PATH
+  export LIBRARY_PATH=$SEISSOL_INSTALL/lib:$LIBRARY_PATH
+  export LD_LIBRARY_PATH=$SEISSOL_INSTALL/lib:$LD_LIBRARY_PATH
+  export PKG_CONFIG_PATH=$SEISSOL_INSTALL/lib/pkgconfig:$PKG_CONFIG_PATH
+  export CMAKE_PREFIX_PATH=$SEISSOL_INSTALL
+  export CPATH=$SEISSOL_INSTALL/include:$CPATH
+  export CPLUS_INCLUDE_PATH=$SEISSOL_INSTALL/include:$CPLUS_INCLUDE_PATH
+  export C_INCLUDE_PATH=$SEISSOL_INSTALL/include:$C_INCLUDE_PATH
+
+2. Load basic modules, source launch file and create an install directory:
+
+::
+
+  $ source ./launch.sh
```

Comment:
Don't you think it would be better to remove the '$' signs?
When you follow such an installation procedure, you just want to copy the code block, not remove every '$', right?

Reply:
Hi, actually  '$' signs help. Firstly, it helps when you want to show a content of a file using `cat` command. It makes clear what is a command and what is its output. For example:

```
$ cat $HOME/launch.sh  
#!/bin/bash  

VERSION="openmpi4-1-1_ucx1-10-1"  
export SEISSOL_INSTALL=$HOME/usr/local/${VERSION}
...
```

Secondly, it can prevent one to blindly copy a command and execute it. You still have a couple of seconds to have a look at and figure out whether you need to adjust it for your environment.

Thirdly, other project adopted this style e.g., LLVM, Docker, Spack, etc.